README.md

```diff
@@ -96,15 +97,22 @@ The `Client` is specific to a configuration and cache file. These approximately
 Available methods on `Client`:
 
 - Projects
-  - get
-  - list
+  - list: Read all Project details.
+  - get: Read Project details.
 - Forms
-  - get
-  - list
+  - list: Read all Form details.
+  - get: Read Form details.
 - Submissions
-  - get
-  - list
-  - get_table
+  - list: Read all Submission metadata.
+  - get: Read Submission metadata.
+  - get_table: Read Submission data.
+  - post: Create a Submission.
+  - put: Update Submission data.
+  - patch: Update Submission metadata.
+  - edit: Submission.post then Comment.post.
```

Comment:
Edit a submission and optionally comment on it.

Reply:
Did you want the above comment to replace all 4 highlighted lines? Or just like `edit: Edit a submission and optionally comment on it.`

Reply:
README reworded per above method name changes